package.json

```diff
@@ -1,38 +1,277 @@
 {
```

Comment:
This seems to have some conflicts in the dependencies, `yarn` fails to install or run. I'm not really sure how to resolve this bc JS, but let's get it to at least run, then we can merge.

Reply:
does the new version still fail?
you might need to run `yarn autoclean --force` after `yarn` off a clean clone, but it should work afterwards or maybe me running it once was enough
if it doesnt work built in I can try adding it to the postinstall script as well